constructor/osxpkg.py

```diff
@@ -276,16 +276,17 @@ def modify_xml(xml_path, info):
     # - Install on a specific disk: /<custom_root>/<name>
     # On modern Mac systems, installing in root is not allowed. So
     # we remove this option by not supplying enable_localSystem
-    # below. Alternatively, we could have chosen not to set the
+    # in the default value for pkg_domains. enable_localSystem
+    # should only be set to true if default_location_pkg is supplied
```

Comment:
Should we add a check in `main.py` testing for this?

Reply:
I added a warning

Reply:
Thanks! Since the incompatibility is already documented, it should be enough (maybe someone reaaally wants to do the wrong thing 🤷).